loginas/utils.py

```diff
@@ -0,0 +1,63 @@
+import logging
+
+from django.conf import settings as django_settings
+from django.contrib.auth.models import User
+from django.contrib.auth import load_backend, login, logout
+from django.contrib import messages
+from django.core.signing import TimestampSigner, SignatureExpired
+from datetime import timedelta
+
+from . import settings
```

Comment:
Oh well it doesn't yet, no I think it's fine to leave it with a clear separation of which is which, django_settings and la_settings, for now :)